internal/commands/app/create.go

```diff
@@ -228,25 +231,34 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 		}
 	}
 
-	if dsCluster.Name != "" {
-		local.AddDataSource(appLocal.AppData, map[string]interface{}{
-			"name": dsCluster.Name,
-			"type": dsCluster.Type,
-			"config": map[string]interface{}{
-				"clusterName":         dsCluster.Config.ClusterName,
-				"readPreference":      dsCluster.Config.ReadPreference,
-				"wireProtocolEnabled": dsCluster.Config.WireProtocolEnabled,
-			},
-		})
+	clusterNames := make([]string, 0, len(dsClusters))
+	for _, dsCluster := range dsClusters {
+		if dsCluster.Name != "" {
+			local.AddDataSource(appLocal.AppData, map[string]interface{}{
+				"name": dsCluster.Name,
+				"type": dsCluster.Type,
+				"config": map[string]interface{}{
+					"clusterName":         dsCluster.Config.ClusterName,
+					"readPreference":      dsCluster.Config.ReadPreference,
+					"wireProtocolEnabled": dsCluster.Config.WireProtocolEnabled,
+				},
+			})
+			clusterNames = append(clusterNames, dsCluster.Config.ClusterName)
+		}
 	}
-	if dsDataLake.Name != "" {
-		local.AddDataSource(appLocal.AppData, map[string]interface{}{
-			"name": dsDataLake.Name,
-			"type": dsDataLake.Type,
-			"config": map[string]interface{}{
-				"dataLakeName": dsDataLake.Config.DataLakeName,
-			},
-		})
+
+	dataLakeNames := make([]string, 0, len(dsDataLakes))
+	for _, dsDataLake := range dsDataLakes {
+		if dsDataLake.Name != "" {
```

Comment:
likewise here, is this condition still applicable?  same comments from above if so